modulith/commodityFinder/src/main/java/io/edpn/backend/modulith/commodityfinder/application/mappers/persistence/StationMapper.java

```diff
@@ -1,7 +1,6 @@
-package io.edpn.backend.modulith.commodityfinder.application.mappers.persistence;
+package io.edpn.backend.modulith.commodityfinder.application.mappers;
```

Comment:
package name and folder are different.... why was this changed?

Reply:
will fix